src/payment/strategies/braintree/braintree-credit-card-payment-strategy.spec.ts

```diff
@@ -161,9 +161,6 @@ describe('BraintreeCreditCardPaymentStrategy', () => {
             expect(braintreePaymentProcessorMock.isInitializedHostedForm).toHaveBeenCalled();
             expect(braintreePaymentProcessorMock.getSessionId).toHaveBeenCalled();
         });
-
-        // throws BraintreeError
```

Comment:
Is this intentional?

Reply:
I forgot to remove the comment in my previous PR 🙈